articles/libraries/machine-learning/load.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Quantum machine learning library
+description: Learn how to load your own dataset to train a classifier model with the Microsoft QDK.
+author: geduardo
+ms.author: v-edsanc@microsoft.com
+ms.date: 02/16/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.machine-learning.load
+---
+
+# Load and classify your own datasets

```

Comment:
There are no actual instructions to load the data in this article, just the date format

Reply:
Good point, added a bit pointing to the samples. Thanks for the catch!